src/style.css

```
@@ -0,0 +1,14 @@
+@import "../styles/reset.css";
+@import "../styles/blocks/card.css";
+@import "../styles/blocks/star.css";
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  color: #060b35;
+  font-size: 10px;
+  line-height: 14px;
+}
```

I`m not sure, but i think that beter to create new file css for set syles in page.Like page.css and give your html-tag class="page".
```suggestion

.page {
  font-family: Roboto, sans-serif;
  color: #060b35;
  font-size: 10px;
  line-height: 14px;
}
```